assets/certs/component-certs/kafka-server-domain.json

```diff
@@ -4,7 +4,11 @@
     "kafka",
     "*.my.domain",
     "kafka.confluent.svc.cluster.local",
-    "*.kafka.confluent.svc.cluster.local"
+    "*.kafka.confluent.svc.cluster.local",
```

Comment:
Thanks for fixing this! We have certificates as part of code in internal repo. Didn't realize that this is separated out here.